pages/common/mpg321.md

```diff
@@ -0,0 +1,33 @@
+# mpg321
+
+> High Performance MPEG 1.0/2.0/2.5 Audio Player for Layer 1, 2, and 3.
+> Mpg321 was written to be a drop-in replacement for the (by-then) non-free mpg123 player.
+> More information: <http://mpg321.sourceforge.net/>.
+
+- Play an audio source (on repeat) exactly N=3 times:
+
+`mpg321 -l {{3}} {{ file(s) | URL }}`
+
+- Play an audio source (on repeat) forever (0 means until interrupted):
+
+`mpg321 -l 0 {{ file(s) | URL }}`
+
+- Play a whole folder (recursively):
+
+`mpg321 -B {{ folder(s) }}`
+
+- Enable Basic Keys ( * or / - Increase or decrease volume, n - Skip song, m - Mute/unmute.) while playing:
```

Comment:
Perhaps this will read better?
```suggestion
- Enable basic keys (`*` or `/`: increase or decrease volume, `n`: Skip song, `m`: Mute / unmute) while playing:
```

Reply:
OK, though Basic Keys is the term used in the documentation...